### ISSUE TITLE
Delete other files from uninstalled addons

### ISSUE THREAD

**Issue by Sekorhex** [WowUp/WowUp#535]
**Is your feature request related to a problem? Please describe.**
When I uninstall an addon, old files still remain. These should be deleted as well.

**Describe the solution you'd like**
It would be great to delete the files in the folder World of Warcraft\_retail_\WTF\Account\0000000#1\SavedVariables when I uninstall an addon with WoWUp.

**Additional context**
The problem with the number could be solved simply by specifying the path of the folder.

**Comment by Tiscan** [comment]
Two other things:

1) I would also like to be able to -just- delete saved variables to 'reset' a mod.

2) Be able to scan for and remove 'orphan' saved variable files so that they don't cause unintended behavior if the mod that uses them is ever reinstalled in the future.

**Comment by jliddev** [comment]
Currently we dont have any plans to mess with the WTF folder but will keep this around for future.

**Comment by Cassiopeia-the-Elf** [comment]
I have just started using WoWUp and this is the main feature that's really missing! Please consider it, at least the 'uninstalling whole addon' part!
Trying out an addon and then uninstalling it would leave a lot of 'trash' in settings folders, especially if you try it on multiple characters. Deleting those files by hand would be such a trouble..

Since many people changed from Twitch to WuWUp after its merge to Overwolf, such a feature would be really appreciated by those who were used to clean uninstallations!
Thank You!

**Comment by Medius7** [comment]
It would be really nice. I think, this would be the exspected behaviour for most users.

**Comment by linaori** [comment]
If implemented it should be a toggle or option when you delete something. I like keeping the files around because I often remove/reinstall addons and don't want to lose my settings.

**Comment by MrUzagi** [comment]
I also would like to see this as an option. Preferably as a checkbox in the dialog as I remove an add-on. The question is how multiple accounts and separate character settings should be handled.

~~It could also be useful to have an additional option of removing the settings without removing the add-on. For the times one need to reset an add-on.~~ Then again, that could be done by reinstalling the add-on as well I suppose. So perhaps not so useful.

**Comment by klepp0906** [comment]
> If implemented it should be a toggle or option when you delete something. I like keeping the files around because I often remove/reinstall addons and don't want to lose my settings.

yep, something akin to what the curseforge client currently offers.  its one of the biggest drawbacks of wowup at current relative to.  actually perhaps the only drawback lol.
